docs/userGuide/syntax/annotations.md

```diff
@@ -0,0 +1,162 @@
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" content="There is only text when you click me" label="1"/>
+  <!-- Set Legend to bottom only (popover is not clickable) -->
+  <a-point x="50%" y="50%" content="Clicking on this does nothing" label="2" legend="bottom" header="Headers are displayed as well"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" content="There is text at both locations"  label="3" legend="both" header="Headers are displayed at both positions"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+**Sample use cases for Annotate** <br>
+
+Here we showcase some use cases of the Annotate feature.
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">html</variable>
+<variable name="code">
+
+<annotate src="../diagrams/activity.png" width="500" alt="Sample Image">
+  <!-- Default Legend (popover only)-->
+  <a-point x="25%" y="50%" content="There is only text when you click me" label="1"/>
+  <!-- Set Legend to bottom only (popover is not clickable) -->
+  <a-point x="50%" y="50%" content="Clicking on this does nothing" label="2" legend="bottom" header="Headers are displayed as well"/>
+  <!-- Set Legend to both -->
+  <a-point x="75%" y="50%" content="There is text at both locations"  label="3" legend="both" header="Headers are displayed at both positions"/>
+</annotate>
+</variable>
+</include>
+
+<br>
+
+****`<a-point>` Options****
+
+| Name      | Type     | Default     | Description                                                                                                                              |
+| --------- | -------- | ----------- | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| x         | `String` |        | **This must be specified.**<br>The x-coordinate of the point.<br>Supports range of values from `0%` to `100%`.           |
+| y         | `String` |        | **This must be specified.**<br>The y-coordinate of the point.<br>Supports range of values from `0%` to `100%`.           |
+| content   | `String` | `''`        | The content to be inserted into the body of the text.<br>The body will be omitted if this is not provided |
+| header    | `String` | `''`        | The content to be inserted into the header of the text.<br>The header will be omitted if this is not provided                            |
+| trigger   | `String` | `click`     | Popover trigger type.<br>Supports: `click`, `focus`, `hover`, or any space-separated combination of these.                       |
+| placement | `String` | `top`       | Position of the Popover.<br>Supports: `top`, `left`, `right`, `bottom`.                                                              |
+| label     | `String` | `''`        | The content to be shown over the point.<br>The label will be omitted if this is not provided                                             |
```

Comment:
Maybe can also include a note to let user know that labels should not be too long to prevent overflow?